Allow proper setCount SLAs across zones (#8752)
Minio-io/minio: docs/distributed/DESIGN.md

> Notice the requirement of common SLA here original cluster had 1024 drives with 16 drives per erasure set, second zone is expected to have a minimum of 16 drives to match the original cluster SLA or it should be in multiples of 16.

> New objects are placed in zones in proportion to the amount of free space in each zone, following pseudo code implements this behavior.

Reviewer: ```suggestion
MinIO places new objects in zones based on proportionate free space, per zone. Following pseudo code demonstrates this behavior.
```
